flood_adapt/object_model/scenario.py

```diff
@@ -73,6 +76,55 @@ def run(self):
                 f"Direct impacts for scenario '{self.attrs.name}' has already been run."
             )
 
+        # Create the infometrics and infographic files
+        self._create_infometrics_infographics()
+
+    def _create_infometrics_infographics(self):
+        # Get the metrics
+        metrics_results_path = self.database_input_path.parent.joinpath(
+            "output",
+            "results",
+            f"{self.attrs.name}",
+            "fiat_model",
+            "output",
+            "output.csv",
+        )
+
+        # Get the metrics configuration
+        metrics_config_path = self.database_input_path.parent.joinpath(
+            "static",
+            "templates",
+            "infometrics",
+            "metrics_config.toml",
+        )
+
+        # Specify the metrics output path
+        metrics_outputs_path = self.database_input_path.parent.joinpath(
+            "output",
+            "infometrics",
+            f"{self.attrs.name}_metrics.csv",
+        )
+
+        # Get the results dataframe
+        df = pd.read_csv(metrics_results_path)
+
+        # Write the metrics to file
+        metrics_writer = MetricsFileWriter(metrics_config_path)
+
+        metrics_writer.parse_metrics_to_file(
+            df_results=df, metrics_path=metrics_outputs_path, write_aggregate=None
+        )
+        metrics_writer.parse_metrics_to_file(
+            df_results=df, metrics_path=metrics_outputs_path, write_aggregate="all"
+        )
+
+        # Get the infographic
+        InfographicsParser().write_infographics_to_file(
```

Comment:
It seems that while the InfographicsParser class is in fiat_toolbox, it is really tailoed to the floodadapt database structure. Wouldn't it make more sense to not rely on this structure since this can be used outside of floodadapt as well? 

I would expect that the infographic method would need the metrics files, and a configuration file only. The database paths should only be defined here in this floodadapt method.